cli/tests/test_split_command.rs

```diff
@@ -226,8 +226,7 @@ fn test_split_with_default_description() {
     std::fs::write(workspace_path.join("file1"), "foo\n").unwrap();
     std::fs::write(workspace_path.join("file2"), "bar\n").unwrap();
 
-    // Create a bookmark pointing to the commit. It will be moved to the second
-    // commit after the split.
+    // Create a bookmark pointing to the commit. It will not move during the split.
```

Comment:
nit: technically, it does move from the old commit to the first split commit. Feel free to either clarify that or leave as is

Reply:
You're right about the technicality. It gets a bit wordy to write it out, but probably better to be precise. Fixed.